doc/module-docs/cc_write_files/example6.yaml

```diff
@@ -0,0 +1,5 @@
+#cloud-config
+write_files:
+- source: https://gitlab.example.com/some_ci_job/artifacts/hello
```

Comment:
This example will need to be updated. Probably best to include both `uri` and `headers`.

Reply:
Ah, I fixed the schema but forgot the example. 🤦